### ISSUE TITLE
Classic citation dialog has buttons not focusable via keyboard

### ISSUE THREAD

**Issue by abaevbog** [zotero/zotero#4708]
Specifically arrows to add/remove seleted items

https://forums.zotero.org/discussion/118213/zotero-7-05-keyboard-functionality-with-classic-view-adding-multiple-sources

**Comment by abaevbog** [comment]
Hm, though the buttons get dynamically disabled/enabled as you navigate through the dialog which means they sometimes are in the taborder, and sometimes not, which is not very good. So it properly fix it, that whole logic would need to be revised. I wonder if it's necessary, since you can use QF to achieve whatever you are after in a more accessible way. Or maybe we can just make them focusable and add aria-labels for now so that it's at least somewhat better. Not sure

**Comment by abaevbog** [comment]
No longer relevant due to the new redesigned citation dialog.
